### ISSUE TITLE
No MQTT dispatch handler for WILDCARD topic

### ISSUE THREAD

**Issue by HolgerAusB** [arachnetech/homebridge-mqttthing#500]
I have two raspberrys working on different corners of my house. One of them is the mosquitto server/dispatcher. Both have a rtl_433 receiver to catch signals of 433 MHz weather-station-sensors and two doorbells. Because of the doorbells with self-havesting energy and low signal, I need the two pi-receivers.

As I want to know, which of my receivers cought a signal, the topc contains `pi1` or `pi2` in the topic, everything else is identical:

```
rtl/pi1/55/77/Acurite-606TX/temperature_C
rtl/pi2/55/77/Acurite-606TX/temperature_C
```
while 55 is the protocol-preset of rtl_433, and 77 the id of the transmitter.

Normally one can subscribe to both topics with topic `rtl/+/55/77/Acurite-606TX/temperature_C` with the `+` as wildcard

But this doesn't work with mqttthing. It does subscribe that wildcard-topic but can't handle the message which contains `pi1` odr `pi2` instead of `+`.

```
Homebridge-Log:
Warning: No MQTT dispatch handler for topic [rtl/pi2/55/84/Acurite-606TX/temperature_C]
```

**Comment by rbswift** [comment]
One solution would be to use something like nodered to subscribe to the wildcard topic and then republish the message on a fixed topic. It would be a very simple flow and nodered is very powerful to support future workflows you might need.

**Comment by HolgerAusB** [comment]
Thank You @rbswift but I will not install a large extra software/server just for fixing a problem of mqttthing. David has applied the "feature request" tag, so I will wait in humility and gratitude.

Meanwhile I took out the server-name out of the topic-path.

**Comment by arachnetech** [comment]
rbswift's suggestion is a good one, but I understand it would be good if this can be implemented within mqttthing to save the additional infrastructure. However, to set your expectations I am only finding time to work on mqttthing occasionally at the moment - so it won't get added immediately...

**Comment by HolgerAusB** [comment]
Yes, I thought so. That's why I wrote '...wait in humility and gratitude'. It is not urgent for me, so I can wait - for as long as it may take.

BTW: Thank you for your work. The plugin was a great help to get my two 433Mhz doorbells working (as motion sensors).

**Comment by tablatronix** [comment]
Ditto, i use tasmota full topics with deviceid, having a topic token would be sweet. I assumed this was automatic already.

**Comment by halk1982** [comment]
Same need here, it seems like mqtthing cannot handle the + wildcard. Any update on this Dave? Thank you very much for your work, this is a great plugin!
